### DISCUSSION TITLE
I lost access to my account with Github Copilot from Student Developer Pack

### DISCUSSION THREAD

**Discussion by clslvs** [community/community#119836] [GitHub Education]
I forgot the recovery codes and my 2FA code won't work at all (I use Google Authenticator) i tried to input it again and again but to no avail it won't register as the code, Do i need to register or verify my student status again? If so, i would like to apply this account as the new holder of Github Copilot

**Comment by Lebohang1821** [comment]
Contact GitHub Support: If you encounter any issues during the recovery process or need further assistance, you can contact GitHub support directly. They can help you with account recovery and any other account-related issues you may have.

Student Status: If your GitHub account is associated with the Student Developer Pack, and you need to verify your student status again, you can follow the process outlined by GitHub for re-verifying your student status. This typically involves reapplying for the Student Developer Pack and providing proof of your current student status.

**Comment by samus-aran** [comment]
Hi 👋 

Are you saying that you are unable to access the GitHub account that has been associated with your academic benefits? 
It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
